h/models/user.py

```diff
@@ -378,4 +379,4 @@ def get_by_username(cls, session, username, authority):
         )
 
     def __repr__(self):
-        return f"<User: {self.username}>"
+        return helpers.repr_(self, ["id", "userid"])
```

Comment:
Username was a useless thing to put in the `__repr__()` of a user: it doesn't uniquely identify the user (two users can have the same username if they have different authorities) and it isn't permanent (you can change your username).

Replaced with the primary key ID and the userid (which is the full, unique `acct:{username}@{authority}`). I'm unsure about including the userid in this: it can change (if you change your username) which could make log messages misleading: you might see a log message with a userid that no longer exists and falsely conclude that the account has been deleted when it was only renamed. Even worse if another account has that username now.